build.sbt

```diff
@@ -1,28 +1,33 @@
-val slickVersion = "3.3.3"
-val akkaVersion = "2.6.19"
-// make sure this is the same as the playWS's dependency
-val playJsonVersion = "2.9.2"
+val slickVersion = "3.4.1"
+val akkaVersion = "2.6.20"
```

Comment:
I believe this needs to be consistent with play's akka, otherwise it would give runtime errors